Author: Otto Kekäläinen <otto@debian.org>
Date:   Thu Jul 25 14:04:04 2019 -0300

    Disable test that apparently has a race condition and sporadically fails.
    Upstream developer Teemu Ollakka confirmed that the failures in this test
    were not a sign of a real bug.

index d579b3e..7d1e248 100644
--- a/galerautils/tests/SConscript
+++ b/galerautils/tests/SConscript
@@ -27,7 +27,6 @@ gu_tests = env.Program(target = 'gu_test
                             gu_fifo_test.c
                             gu_uuid_test.c
                             gu_dbug_test.c
-                            gu_lock_step_test.c
                             gu_str_test.c
                             gu_utils_test.c
                          '''))
--- a/galerautils/tests/gu_tests.c
+++ b/galerautils/tests/gu_tests.c
@@ -20,7 +20,6 @@
 #include "gu_time_test.h"
 #include "gu_fifo_test.h"
 #include "gu_uuid_test.h"
-#include "gu_lock_step_test.h"
 #include "gu_str_test.h"
 #include "gu_utils_test.h"
 
@@ -39,7 +38,6 @@ static suite_creator_t suites[] =
         gu_time_suite,
         gu_fifo_suite,
         gu_uuid_suite,
-        gu_lock_step_suite,
         gu_str_suite,
         gu_utils_suite,
         NULL
--- a/gcs/src/unit_tests/SConscript
+++ b/gcs/src/unit_tests/SConscript
@@ -45,7 +45,6 @@ gcs_tests_sources = Split('''
                              gcs_memb_test.cpp
                              ../gcs_code_msg.cpp
                              ../gcs_group.cpp
-                             gcs_core_test.cpp
                              ../gcs_core.cpp
                              ../gcs_dummy.cpp
                              ../gcs_msg_type.cpp
--- a/gcs/src/unit_tests/gcs_tests.cpp
+++ b/gcs/src/unit_tests/gcs_tests.cpp
@@ -21,7 +21,6 @@
 #include "gcs_act_cchange_test.hpp"
 #include "gcs_group_test.hpp"
 #include "gcs_backend_test.hpp"
-#include "gcs_core_test.hpp"
 #include "gcs_fc_test.hpp"
 
 typedef Suite *(*suite_creator_t)(void);
@@ -39,7 +38,6 @@ static suite_creator_t suites[] =
 	gcs_act_cchange_suite,
 	gcs_group_suite,
 	gcs_backend_suite,
-	gcs_core_suite,
 	gcs_fc_suite,
 	NULL
     };
